tests/index.js

```diff
@@ -0,0 +1,14 @@
+import chai from 'chai';
+import sinonChai from 'sinon-chai';
+import sinon from 'sinon/pkg/sinon-esm';
+// import 'promise-polyfill/src/polyfill';
```

Comment:
redundant

Reply:
Right , mobed it to there and forgot  to remove it here